### PR TITLE
Complete test metrics and plots, tune learning rate, etc.

### PR DESCRIPTION
**Pull request by J22Melody**
Confusion matrix, pr curve, and probs plots:

https://wandb.ai/zifanjiang/pose-to-segments/runs/tvxti7jt?workspace=user-j22melody

Tune learning rate - seems a bigger learning rate helps, but `ReduceLROnPlateau` does not:

- 10e-3: https://wandb.ai/zifanjiang/pose-to-segments/runs/9su73aq7?workspace=user-j22melody
- 10e-4: https://wandb.ai/zifanjiang/pose-to-segments/runs/iw7njj9q?workspace=user-j22melody
- 10e-3 with `ReduceLROnPlateau`: https://wandb.ai/zifanjiang/pose-to-segments/runs/sc4ld8p2?workspace=user-j22melody

### CODE REVIEW

**Review by AmitMY — APPROVED** [review]
lgtm, and I saw the figures on wandb. looks like you've been doing lots of work on this!

Did you have a training run for a long while that also has figures? I'm interested in seeing them.

**Comment by J22Melody** [comment]
@AmitMY ahh I forgot to save the PR's descriptions, now you see it, yes and the figures on wandb.

These newly added figures (confusion matrix, pr curve, probs plots) are only generated in the end once for the test data. I am not plotting them during training because that would be many. Do you wanna see how they change during training? One possible way would be to plot them every 10 epochs or so.

Now it also reminds me that I should probably not plot on test data and tune the model accordingly, I should use the dev data instead!
